client/components/place/place-list/place-list.template.html

```diff
@@ -298,81 +296,200 @@
                    ng-model="$ctrl.newPlacePhoto">
           </div>
 
-          <div class="form-group">
+          <div class="form-group visible-lg">
```

Comment:
It is standard bootstrap style which shows current block only on large screens and hides it on another screen types.